### PR TITLE
fix redis implementation

### PR DESCRIPTION
**Pull request by rwaffen**
* add ddl from R.I.Pienaar for redis
* remove duplicated parameter from connector

### CODE REVIEW

File: files/plugins/connector/redis/mcollective/connector/redis.ddl [hunk 1]
```diff
@@ -0,0 +1,9 @@
+metadata    :name        => "Redis Connector",
+            :description => "Connector plugin for Redis middleware",
+            :author      => "R.I.Pienaar <rip@devco.net>",
+            :license     => "ASL 2.0",
+            :version     => "1.0.0",
+            :url         => "https://github.com/ripienaar/mcollective-vagrant",
```
Comment (bastelfreak) [inline files/plugins/connector/redis/mcollective/connector/redis.ddl]: Is this url correct? It does not seem to be related to the redis connector.

**Comment by bastelfreak** [comment]
Hi @rwaffen, thanks for this PR! Is it possible to add a simple unit test somehow for this change? (It is okay if not)

File: files/plugins/connector/redis/mcollective/connector/redis.ddl (hunk 1 above)
Comment (rwaffen) [reply files/plugins/connector/redis/mcollective/connector/redis.ddl]: I couldn't find another or "the" source, but the redis connector in both  mcollective modules are the same. So the voxpupuli one and the one from https://github.com/ripienaar/mcollective-vagrant/tree/master/deploy/modules/mcollective/files/lib/connector are the same.

this is only a meta file, but i don't wanted to write my name in there, i only wanted it to work ;-)

**Comment by rwaffen** [comment]
i will check, if i'm able to write a test. I'm not used to write rspec tests for other things than puppet...

**Comment by rwaffen** [comment]
hi there, i don't get this running with unit tests. sry for that. 😞
